ops/ansible/playbooks-healthapt/backend.yml

```diff
@@ -116,6 +116,16 @@
         mode: 'u=rw,g=,o='
       become: true
 
+    # Overwrite the bfd-server role generation of the keystore file with the one in the ansible vault 
+    - name: Copy the appserver keystore  
```

Comment:
For safety's sake, I think you need to copy the current keystore to a backup file here, first. Would be better than trying to rummage through EC2 snapshots for the keystore, if we needed to rollback. (Too late for our TEST environment, but that's not a big deal.)